src/tests/Common/XUnitWrapperLibrary/TestSummary.cs

```diff
@@ -133,8 +140,10 @@ public void WriteFooterToTempLog(StreamWriter tempLogSw)
         var result = new TestResult(name, containingTypeName, methodName, duration, null, null, output);
         _testResults.Add(result);
 
+        outTw.WriteLine($"{0:HH:mm:ss.fff} Passed test: {1}", System.DateTime.Now, name);
```

Comment:
@markples This line results in printing "HH:mm:ssfff Passed test: 1" always -- presumably it shouldn't be a string interpolation.

Reply:
Ah, that's leftover from the original
```
builder.AppendLine($"System.Console.WriteLine(\"{{0:HH:mm:ss.fff}} Running test: {{1}}\", System.DateTime.Now, {test.TestNameExpression});");
```
but was for the outer level. It's weird though; I thought that I looked at the output.